scripts/export.js

```diff
@@ -61,22 +52,14 @@ try {
     // quadrant ordering may be non-deterministic.
     quadrants.sort((a, b) => quadrantSortValues[a] - quadrantSortValues[b]);
     quadrants.forEach(quadrant => {
-      const ringsPath = path.resolve(quadrantsPath, quadrant, 'rings');
-      const rings = fs.readdirSync(ringsPath);
-      // This sorts the rings in place according to the ordering above.  Without this our
-      // ring ordering may be non-deterministic.
-      rings.sort((a, b) => ringSortValues[a] - ringSortValues[b]);
-      rings.forEach(ring => {
-        const blipsPath = path.resolve(ringsPath, ring, 'blips');
-        const blips = fs.readdirSync(blipsPath);
-        // This will just sort the blips alphabetically.  Whether this is the right ordering is up
-        // for debate.
-        blips.sort();
-        blips.forEach(blip => {
-          const blipPath = path.resolve(blipsPath, blip);
-          const blipData = require(blipPath);
-          radarJson.push(blipData);
-        });
+      const blipsPath = path.resolve(quadrantsPath, quadrant, 'blips');
+      const blips = fs.readdirSync(blipsPath);
+      // This will sort the blips alphabetically, regardless of ring.
+      blips.sort();
```

Comment:
Hmm.  I think there's likely a possible bug here.  The order of rings in the tech radar visualization is based on their appearance in the CSV.  If the first blip alphabetically happens to be "Provisional", for example, that will make Provisional the innermost ring.  Adding blips by ring had the benefit of ensuring that the rings were added in the right order.

The ring information is in the blips themselves - I think we could add a compare function to the sort here and use an object like `ringSortValues` to ensure that the blips are ordered first by ring and _then_ alphabetically.  

Something like:

```
const ringSortValues = {
  adopted: 0,
  accepted: 1,
  provisional: 2,
  hold: 3,
};

blips.sort((a, b) => ringSortValues[a.ring] - ringSortValues[b.ring]);
```

That'll preserve the right behavior, I think.  Note: I didn't test that code.

Reply:
Should they be ordered first by ring, and then alphabetically by blip name? Or should they be ordered by quadrant as well? If so, how?